user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -58,6 +58,7 @@ class QualificationSummary:
     recommended_apps: pd.DataFrame = None
     df_result: pd.DataFrame = None
     irrelevant_speedups: bool = False
+    pricing_config: Any = None
```

Comment:
is `pricing_config` is only used as a flag? or this is this defined as any to support another feature in the upcoming PR?

Reply:
It is used as a flag. If the pricing information is not available in config file, then we run qualification tool based on speedups and do not calculate/display cost savings.